dts/xtensa/intel/intel_adsp_ace15_mtpm.dtsi

```diff
@@ -313,6 +313,14 @@
 			};
 		};
 
+		ace_comm_widget: comm_widget@71C00 {
```

Comment:
nit: `ace_comm_widget:` -> `comm_widget:`

Reply:
what does "nit" mean?
Please be more specific.

Reply:
Sorry, nit is short for nitpick.  (nitpick definition is: engage in fussy or pedantic fault-finding).  So using it to say that the comment is a pretty minor detail.